dbtmetabase/__init__.py

```diff
@@ -120,99 +173,133 @@ def main(args: List = None):
     )
 
     parser = argparse.ArgumentParser(
-        description="Model synchronization from dbt to Metabase."
+        prog="PROG", description="Model synchronization from dbt to Metabase."
     )
-    parser.add_argument("command", choices=["export"], help="command to execute")
 
-    # dbt arguments
+    # Commands
     parser.add_argument(
+        "command", choices=["models", "exposures"], help="Command to execute"
+    )
+
+    parser_dbt = parser.add_argument_group("dbt Parser")
+    parser_metabase = parser.add_argument_group("Metabase Client")
+    parser_models = parser.add_argument_group("`models` optional arguments")
+    parser_exposures = parser.add_argument_group("`exposures` optional arguments")
+
+    # dbt arguments
```

Comment:
@z3z1ma It looks like `--schema` was dropped from this list of arguments. Since it has the default of public there's not an error message. I think this in combination with [how I was invoking](https://github.com/gouline/dbt-metabase/pull/38#issuecomment-885759913) is the root cause of the missing exposures.